dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/transform/strategy/PageViewStrategy.java

```diff
@@ -32,6 +38,7 @@
     PageViewStrategy(final TransformToolbox toolBox) {
         super(toolBox);
         htmlPageCache = CacheLocator.getHTMLPageCache();
+        permissionAPI = APILocator.getPermissionAPI();
```

Comment:
I know I broke my own rule here. But I think permissionAPI should probably be in TransformToolbox for the sake of being able to feed all services with mocks from one singe entry point,

Reply:
The Toolbox was generalized to be used by any class. Also it was refactored to use the Builder pattern to ease the use of it.